### PR TITLE
feat(workspace) allow to expand the workspace to 500px height

### PR DESCRIPTION
**Pull request by pelord**
**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines:https://github.com/infra-geo-ouverte/igo2/blob/master/.github/CONTRIBUTING.md#git-commit-guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
The workspace table is restricted to 285 pixels

**What is the new behavior?**
Allow to expand the workspace to 500px height

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [x] No

**If this PR contains a breaking change, please describe the impact and migration path for existing applications:**

**Other information**:

### CODE REVIEW

File: packages/integration/src/lib/workspace/workspace.state.ts [hunk 1]
```diff
@@ -74,6 +81,14 @@ export class WorkspaceState implements OnDestroy {
       });
     });
 
+    this.actionFullExtent$$.push(this.featureActionsService.getWorkspaceFullExtentEmitter().subscribe(r => {
```
Comment (cbourget) [inline packages/integration/src/lib/workspace/workspace.state.ts:84]: Shouldn't this be the other way around? The action services should be stateless and hte "set full extent" action should modify/observe the workspace state.

**Review by cbourget — COMMENTED** [review]
I find extent confusing because we generally associate extent with the map extent. Also, see my inline comment.

**Comment by pelord** [comment]
> I find extent confusing because we generally associate extent with the map extent. Also, see my inline comment.

Has been changed to maximize.

File: packages/integration/src/lib/workspace/workspace.state.ts (hunk 1 above)
Comment (pelord) [reply packages/integration/src/lib/workspace/workspace.state.ts]: Could you take a look to this new version?

**Review by mbarbeau — APPROVED** [review]

**Pull request merged**